src/cdk/datepicker/datepicker.md

```diff
@@ -0,0 +1 @@
+Will write this after completion of integration between material datepicker and CDK datepicker.
```

Comment:
you can delete this file and add it back when there's something useful to put here